Formula/crystal.rb

```diff
@@ -96,8 +96,8 @@ def install
 
     # Build shards
     resource("shards").stage do
-      system buildpath/"boot/embedded/bin/shards", "install",
-                                                   "--production"
+      shard = OS.mac? ? buildpath/"boot/embedded/bin/shards" : buildpath/"boot/bin/shards"
+      system shard, "install", "--production"
 
       system buildpath/"bin/crystal", "build",
                                       "-o", buildpath/".build/shards",
```

Comment:
@iMichka can you check if using `--threads=1` helps the build?

```suggestion
                                      "-o", buildpath/".build/shards",
                                      "--threads=1"
```

Reply:
hmmm, the paste bien shows `--threads=1` is been used, yet it's not stated here 🤔

Reply:
I started a new build with this flag.